proto/osmosis/twap/v1beta1/query.proto

```diff
@@ -18,18 +18,16 @@ service Query {
   rpc Params(ParamsRequest) returns (ParamsResponse) {
     option (google.api.http).get = "/osmosis/twap/v1beta1/Params";
   }
-  rpc GetArithmeticTwap(GetArithmeticTwapRequest)
-      returns (GetArithmeticTwapResponse) {
-    option (google.api.http).get = "/osmosis/twap/v1beta1/GetArithmeticTwap";
+  rpc ArithmeticTwap(ArithmeticTwapRequest) returns (ArithmeticTwapResponse) {
```

Comment:
since we're using a template to generate grpc queries, i think it'd be better if we have a consistent naming standard 

For instance, Gamm queries start with "query" prefix like QueryPoolsRequest QueryTotalLiquidityRequest QuerySpotPriceResponse

However, Twap queries doesnot contain "query" prefix like ArithmeticTwapRequest,ArithmeticTwapToNowRequest,

Reply:
Good point, the Query prefix is standard within the SDK as well

Reply:
This decision has no state machine or client facing affects as far as I'm aware, so lets decide in an issue?

Reply:
Gotchu!